clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/deploy/ops/loadbalancer/UpsertKubernetesLoadBalancerAtomicOperation.groovy

```diff
@@ -63,7 +63,7 @@ class UpsertKubernetesLoadBalancerAtomicOperation implements AtomicOperation<Map
 
     serviceBuilder = serviceBuilder.withNewMetadata().withName(name)
 
-    def labels = existingService?.metadata?.labels ?: [:]
+    def labels = description.serviceLabels ?: existingService?.metadata?.labels
```

Comment:
This prevents me from clearing labels on an existing load balancer, right? (i.e., I get the existing service's labels if `description.serviceLabels` is `null` or `[:]`)